php/class-controller-assets.php

```diff
@@ -22,7 +22,10 @@ function __construct() {
 	 * Do Work
 	 */
 	public function run() {
-		add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue_scripts' ) );
+		global $pagenow;
```

Comment:
Hey @m9n looking good, but a couple of code quality things:

- Ensure globals are at the top of the function.
- Use !== instead of != 
- Use 'Yoda' type expressions, so `if ( 'media-new.php' !== $pagenow  )`

The code sniffer should pick these up in your IDE, if they aren't let me know and we can work out why these rules are not triggering.